# specify sorted signals to require in order for a file to be included
signals_required = [
    "AccelerationX",
    "AccelerationY",
    "AccelerationZ",
    "Altitude",
    "AltitudeAccuracy",
    "AltitudeValid",
    "AngularRateX",
    "AngularRateY",
    "AngularRateZ",
    "AttitudeValid",
    "BMSIgnition",
    "BMSMainRelay",
    "BatteryAvailableChargePower",
    "BatteryAvailableDischargePower",
    "BatteryCurrent",
    "BatteryDCVoltage",
    "BatteryFanFeedback",
    "BatteryFanStatus",
    "BatteryHeaterTemperature1",
    "BatteryMaxTemperature",
    "BatteryMinTemperature",
    "BatteryTemperature1",
    "BatteryTemperature2",
    "BatteryTemperature3",
    "BatteryTemperature4",
    "BatteryTemperature5",
    "BatteryVoltageAuxillary",
    "CCC_CumulativeChargeCurrent",
    "CDC_CumulativeDischargeCurrent",
    "CEC_CumulativeEnergyCharged",
    "CED_CumulativeEnergyDischarged",
    "Charging",
    "DistanceAccuracy",
    "DistanceTotal",
    "DistanceTrip",
    "DistanceValid",
    "Epoch",
    "Fence1",
    "Fence2",
    "Fence3",
    "Fence4",
    "FenceCombined",
    "FenceValid",
    "FixType",
    "Heading",
    "HeadingAccuracy",
    "ImuValid",
    "IndoorTemperature",
    "Latitude",
    "Longitude",
    "MaxCellVoltage",
    "MaxCellVoltageCellNo",
    "MinCellVoltage",
    "MinCellVoltageCellNo",
    "MinDeterioration",
    "MinDeteriorationCellNo",
    "NormalChargePort",
    "OperatingTime",
    "OutdoorTemperature",
    "Pitch",
    "PitchAccuracy",
    "PositionAccuracy",
    "PositionValid",
    "RapidChargePort",
    "Roll",
    "RollAccuracy",
    "Satellites",
    "Speed",
    "SpeedAccuracy",
    "SpeedValid",
    "StateOfChargeBMS",
    "StateOfChargeDisplay",
    "StateOfHealth",
    "TimeConfirmed",
    "TimeValid",
    "TirePressureBackLeft",
    "TirePressureBackRight",
    "TirePressureFrontLeft",
    "TirePressureFrontRight",
    "TireTemperatureBackLeft",
    "TireTemperatureBackRight",
    "TireTemperatureFrontLeft",
    "TireTemperatureFrontRight",
]
